### PR TITLE
chore: do not override RSTUDIO_VERSION in CI

### PR DESCRIPTION
**Pull request by rokroskar**
No double-override.

### CODE REVIEW

**Comment by rokroskar** [comment]
@leafty yes sure, that sounds good. I'm not really up to speed on how to fix these tests so some help would be appreciated there...

**Comment by leafty** [comment]
tests may be a bit flaky now, but they should ultimately pass

**Comment by rokroskar** [comment]
Sorry @leafty I missed that you had updated the tests here. Thanks for that! @olevski could you please review when you get a chance?

File: tests/package.json [hunk 1]
```diff
@@ -9,11 +9,11 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@renku/notebooks-cypress-tests": "^0.0.7",
```
[the remark this answers is not in the document]
Comment (olevski) [reply tests/package.json:12]: yes

[the remark this answers is not in the document]
Comment (leafty) [reply tests/package.json:12]: I don't have any permissions to publish the fix on npm. Can that be approved and published?

File: tests/package.json [hunk 2]
```diff
@@ -9,11 +9,11 @@
...
+    "@renku/notebooks-cypress-tests": "github:leafty/notebooks-cypress-tests#gh-package-11",
```
Comment (olevski) [inline tests/package.json:12]: The comment about using the regular version refers to this... Not to the removal.

Comment (rokroskar) [reply tests/package.json:12]: aha! thanks, that makes more sense now

File: tests/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (rokroskar) [reply tests/package.json:12]: @leafty @olevski what needs to happen to get this through? I will add @leafty to the npm org

File: tests/package.json (hunk 2 above)
Comment (leafty) [inline tests/package.json:12]:
```suggestion
    "@renku/notebooks-cypress-tests": "^0.0.11",
```

File: tests/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (leafty) [reply tests/package.json:12]: Done

[the remark this answers is not in the document]
Comment (rokroskar) [reply tests/package.json:12]: great, thanks!!

**Review by olevski — APPROVED** [review]

**Pull request merged**